README.md

```diff
@@ -1 +1,43 @@
-# Math-Magicians
\ No newline at end of file
+# Math Magicians
+This is a calculator application. The user can do some calculations on this application.
+
+## Built with:
+ * Reactjs
+ * Redux
+
+## Live 
+[Live demo deployed on Netlify](https://mymath-magicians.netlify.app/)
+
+## Usage
+To clone the repo:
+
+$ git clone git@github.com:fed1k/Math-Magicians.git
+
+
+To build the project run:
```

Comment:
[OPTIONAL]

- After cloning you need to mention the step `cd Math-Magicians` to enter into the correct folder. Please do that before merging.

Reply:
@Dipeshtwis Hello bro. Thank you for reviewing and approving. I just implemented this change. Now I'm going to merge this branch. Thanks again. Have a nice time.